.circleci/config.yml

```
@@ -0,0 +1,71 @@
+orbs:
+  architect: giantswarm/architect@0.4.3
```

```suggestion
  architect: giantswarm/architect@0.4.5
```

@calvix created this earlier for pushing to Aliyun. Docs PR is up.